From 78147e20209debb1cd627c37fc2f9d5798482bef Mon Sep 17 00:00:00 2001
From: snowman2 <alansnow21@gmail.com>
Date: Thu, 15 Feb 2024 22:40:25 -0600
Subject: [PATCH] BUG: Support azimuthal_equidistant coordinate operation to CF

---
 pyproj/crs/_cf1x8.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproj/crs/_cf1x8.py b/pyproj/crs/_cf1x8.py
index e58c7002..fbe53547 100644
--- a/pyproj/crs/_cf1x8.py
+++ b/pyproj/crs/_cf1x8.py
@@ -658,6 +658,7 @@ def _pole_rotation_netcdf__to_cf(conversion):
 _INVERSE_GRID_MAPPING_NAME_MAP = {
     "albers_equal_area": _albers_conical_equal_area__to_cf,
     "modified_azimuthal_equidistant": _azimuthal_equidistant__to_cf,
+    "azimuthal_equidistant": _azimuthal_equidistant__to_cf,
     "geostationary_satellite_(sweep_x)": _geostationary__to_cf,
     "geostationary_satellite_(sweep_y)": _geostationary__to_cf,
     "lambert_azimuthal_equal_area": _lambert_azimuthal_equal_area__to_cf,
