From c51fd2c56dde3c7487070d19d360fcb792d7f099 Mon Sep 17 00:00:00 2001
From: snowman2 <alansnow21@gmail.com>
Date: Tue, 20 Dec 2022 21:13:48 -0600
Subject: [PATCH] TST: Fix expected tranform grid list for bbox

---
 test/test_sync.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/test/test_sync.py b/test/test_sync.py
index c0b3cd368..0cca6483c 100644
--- a/test/test_sync.py
+++ b/test/test_sync.py
@@ -31,7 +31,6 @@ def test_get_transform_grid_list__bbox__antimeridian():
     assert sorted(source_ids) == [
         "au_ga",
         "nc_dittt",
-        "no_kv",
         "nz_linz",
         "us_nga",
         "us_noaa",
@@ -50,7 +49,6 @@ def test_get_transform_grid_list__bbox__out_of_bounds():
     assert sorted(source_ids) == [
         "au_ga",
         "nc_dittt",
-        "no_kv",
         "nz_linz",
         "us_nga",
         "us_noaa",
