diff -ruN pyproj-3.6.1/test/conftest.py pyproj-3.6.1-patch/test/conftest.py
--- pyproj-3.6.1/test/conftest.py	2023-09-21 03:30:44.000000000 +0200
+++ pyproj-3.6.1-patch/test/conftest.py	2024-07-28 21:45:37.939141488 +0200
@@ -19,6 +19,7 @@
 PROJ_GTE_92 = PROJ_LOOSE_VERSION >= version.parse("9.2.0")
 PROJ_GTE_921 = PROJ_LOOSE_VERSION >= version.parse("9.2.1")
 PROJ_GTE_93 = PROJ_LOOSE_VERSION >= version.parse("9.3.0")
+PROJ_GTE_941 = PROJ_LOOSE_VERSION >= version.parse("9.4.1")
 
 
 def unset_data_dir():
diff -ruN pyproj-3.6.1/test/crs/test_crs.py pyproj-3.6.1-patch/test/crs/test_crs.py
--- pyproj-3.6.1/test/crs/test_crs.py	2023-09-21 03:30:44.000000000 +0200
+++ pyproj-3.6.1-patch/test/crs/test_crs.py	2024-07-28 21:48:46.218388751 +0200
@@ -25,11 +25,11 @@
     PROJ_GTE_901,
     PROJ_GTE_911,
     PROJ_GTE_921,
+    PROJ_GTE_941,
     assert_can_pickle,
     grids_available,
 )
 
-
 class CustomCRS:
     def to_wkt(self):
         return CRS.from_epsg(4326).to_wkt()
@@ -649,17 +649,15 @@
     assert grid.short_name == "ca_nrc_ntv1_can.tif"
     assert grid.package_name == ""
     assert grid.url == "https://cdn.proj.org/ca_nrc_ntv1_can.tif"
-    if (PROJ_GTE_91 and grids_available(grid.short_name, check_network=False)) or (
-        not PROJ_GTE_91 and grids_available(grid.short_name)
-    ):
+    if grids_available(grid.short_name, check_network=False):
         assert grid.available is True
         assert grid.full_name.endswith(grid.short_name)
-    elif PROJ_GTE_911 and pyproj.network.is_network_enabled():
-        assert grid.available is True
-        assert grid.full_name == grid.url
-    elif PROJ_GTE_91 and pyproj.network.is_network_enabled():
+    elif pyproj.network.is_network_enabled():
         assert grid.available is True
-        assert grid.full_name == ""
+        if PROJ_GTE_941:
+            assert grid.full_name == ""
+        else:
+            assert grid.full_name == grid.url
     else:
         assert grid.available is False
         assert grid.full_name == ""
