From cdde16538b85cca3887478a1ffc30269a083538e Mon Sep 17 00:00:00 2001
From: snowman2 <alansnow21@gmail.com>
Date: Fri, 30 Aug 2024 07:30:06 -0500
Subject: [PATCH] TST: Fix transformer area of interest tests

---
 test/test_transformer.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/test_transformer.py b/test/test_transformer.py
index ed1f231b..629a936a 100644
--- a/test/test_transformer.py
+++ b/test/test_transformer.py
@@ -745,7 +745,7 @@ def test_transformer__area_of_interest():
     transformer = Transformer.from_crs(
         "EPSG:7789",
         "EPSG:4936",
-        area_of_interest=AreaOfInterest(-177.25, -44.64, -43.3, -175.54),
+        area_of_interest=AreaOfInterest(-177.25, -44.64, -175.54, -43.3),
     )
     assert (
         transformer.description
@@ -757,7 +757,7 @@ def test_transformer_proj__area_of_interest():
     transformer = Transformer.from_proj(
         "EPSG:7789",
         "EPSG:4936",
-        area_of_interest=AreaOfInterest(-177.25, -44.64, -43.3, -175.54),
+        area_of_interest=AreaOfInterest(-177.25, -44.64, -175.54, -43.3),
     )
     assert (
         transformer.description
