--- pyproj-1.9.5.1rel.orig/setup.py	2016-01-06 20:26:39.000000000 -0200
+++ pyproj-1.9.5.1rel/setup.py	2018-09-18 11:57:46.088902563 -0300
@@ -24,13 +24,12 @@
     runtime_library_dirs=libdirs,libraries=libraries)
 
     # over-write default data directory.
-    pyproj_datadir = os.path.join(os.path.join(proj_dir,'share'),'proj')
-    datadirfile = os.path.join('lib',os.path.join('pyproj','datadir.py'))
-    datadirfile_save = os.path.join('lib',os.path.join('pyproj','datadir.py.save'))
+    datadirfile = os.path.join('lib',os.path.join('pyproj', 'datadir.py'))
+    datadirfile_save = os.path.join('lib',os.path.join('pyproj', 'datadir.py.save'))
     if not os.path.isfile(datadirfile_save):
         shutil.copyfile(datadirfile, datadirfile_save)
     f = open(datadirfile,'w')
-    f.write('pyproj_datadir="%s"\n' % pyproj_datadir)
+    f.write("import os; pyproj_datadir = os.environ['PROJ_LIB']")
     f.close()
 
     extensions = [pyprojext]
